### PR TITLE
Admin lte update

### LINKED ISSUE #1178: Update Admin LTE  to latest version
Fixes #1178

### PR DESCRIPTION
**Pull request by ashu555**
Fixes #1178 
Admin LTE updated to latest version

### CODE REVIEW

File: app/Http/Controllers/Auth/BaseAuthController.php [hunk 1]
```diff
@@ -104,6 +104,7 @@ public function requestOtpFromAjax(Request $request)
         User::where('email', $email)->update(['email'=>$newEmail, 'mobile'=>$newNumber]);
 
         try {
+            dd($request->input('code'));
```
Comment (mverma16) [inline app/Http/Controllers/Auth/BaseAuthController.php:107]: remove this live dd.

File: app/Http/Controllers/Common/SettingsController.php [hunk 2]
```diff
@@ -384,15 +384,7 @@ public function getMails()
...
-                               $date = $model->date;
-                               if ($date) {
-                                   $date1 = new \DateTime($date);
-                                   $tz = \Auth::user()->timezone()->first()->name;
-                                   $date1->setTimezone(new \DateTimeZone($tz));
-                                   $finalDate = $date1->format('M j, Y, g:i a ');
-                               }
-
-                               return $finalDate;
+                               return getDateHtml($model->date);
```
Comment (mverma16) [inline app/Http/Controllers/Common/SettingsController.php:387]: why controller changes are part of theme update PR?

File: routes/web.php [hunk 3]
```diff
@@ -226,6 +226,7 @@
         Route::get('get-state/{state}', 'Payment\TaxController@getState');
         Route::get('get-tax', ['as' => 'get-tax', 'uses' => 'Payment\TaxController@getTax']);
         Route::get('get-loginstate/{state}', 'Auth\AuthController@getState');
+        Route::get('get-loginstate/{state}', 'Auth\AuthController@getState');
```
Comment (mverma16) [inline routes/web.php:229]: what's this?

File: resources/views/vendor/log-viewer/dashboard.blade.php [hunk 4]
```diff
@@ -13,7 +13,7 @@
       </ol>
       @stop
 @section('content')
```
Comment (mverma16) [inline resources/views/vendor/log-viewer/dashboard.blade.php:15]: should not be part of the PR as does not contain any change except adding/removing whitespaces

File: app/Http/Controllers/Auth/BaseAuthController.php (hunk 1 above)
Comment (ashu555) [reply app/Http/Controllers/Auth/BaseAuthController.php:107]: Fixed

File: app/Http/Controllers/Common/SettingsController.php (hunk 2 above)
Comment (ashu555) [reply app/Http/Controllers/Common/SettingsController.php:387]: Reverted. Will make this part of other PR.

File: routes/web.php (hunk 3 above)
Comment (ashu555) [reply routes/web.php:229]: Removed. Probably occured while rebasing

File: resources/views/vendor/log-viewer/dashboard.blade.php (hunk 4 above)
Comment (ashu555) [reply resources/views/vendor/log-viewer/dashboard.blade.php:15]: reverted

**Comment by mverma16** [comment]
Approving as the impact of changes can not be predicted by code review.

As it contains theme update changes for various js and css files there are possibilities of random issues on random pages.

Testers do co-ordinate with @ashu555 and instruct him what report do you need before you start testing the PR. Do not test before you get required updates and report from @ashu555.

**Comment by bindu95** [comment]
As we(@ladybirdweb/lws-quality-analyst-team) are occupied with the helpdesk releases and also the planned tasks for helpdesk keeping this pr on hold as it is completely UI related and not functionality.

**Comment by bindu95** [comment]
If any suggestions are there, please let us know

**Comment by bindu95** [comment]
> Approving as the impact of changes can not be predicted by code review.
> 
> As it contains theme update changes for various js and css files there are possibilities of random issues on random pages.
> 
> Testers do co-ordinate with @ashu555 and instruct him what report do you need before you start testing the PR. Do not test before you get required updates and report from @ashu555.
> 
> Keep the expectations clear to avoid disputes later.

Sure sir @mverma16  before starting we will check with @ashu555

**Comment by bhanu2217** [comment]
One round of testing and bug fixing is done for this. 2nd round of testing is pending.

**Review by SarfrazS — APPROVED** [review]
I checked all curd and functionality working, some issues are created separately.
I am approving this PR but some issues are fixed on another PR  Taxes PR.
